### PR TITLE
Add bottom tab navigation, and navbar links for web

### LINKED ISSUE #144: Tab bar navigation
_As a user, I want to be able to switch between Search and Bookings (MMB) in the app._ 

Let's implement a simple tab bar navigation on the bottom of the screen. 

For the initial phase, there would be only two tabs: Search and Bookings. Please, align the icons properly :-) 

snimek obrazovky 2019-01-24 v 14 06 28

Design in Figma (for now, use only the Tab bar part): https://www.figma.com/file/GpBUlzILpGgkmmbv4oItD2ZD/Menu-(profile)?node-id=179%3A3011

### PR DESCRIPTION
**Pull request by tbergq**
closes #144

### CODE REVIEW

File: apps/web/components/Navbar.js [hunk 1]
```diff
@@ -39,7 +43,7 @@ const styles = StyleSheet.create({
     },
     shadowRadius: 3,
     paddingHorizontal: 10,
-    justifyContent: 'flex-start',
+    justifyContent: 'space-between',
```
Comment (RobinCsl) [inline apps/web/components/Navbar.js]: I am not entirely sure about this layout, but I guess we would need designs to tell us what we should do.

Comment (tbergq) [reply apps/web/components/Navbar.js]: It is how the web navbar is
screenshot 2019-01-25 at 10 04 11

File: apps/web/components/NavbarLink.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+|};
+
+export default class NavbarLink extends React.Component<Props, State> {
+  state = {
+    isActive: false,
+  };
+
+  componentDidMount() {
+    if (Router.route === this.props.route) {
+      this.onMouseEnter();
```
Comment (FilipMessa) [inline apps/web/components/NavbarLink.js]: Just one think this naming `this.onMouseEnter();` is confusing a little bit.
probably just will be more clearly: 
```
 this.setState({ isActive: true });
```

File: apps/web/components/NavbarLink.js [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+  };
+
+  onPress = () => {
+    Router.push(this.props.route);
+  };
+
+  render() {
+    return (
+      <TouchableWithoutFeedback
+        onMouseEnter={this.onMouseEnter}
```
Comment (JoseDuSV) [inline apps/web/components/NavbarLink.js]: Can we please use `Hoverable` from `universal-components` instead?
```
<Hoverable
   onMouseEnter={this.onMouseEnter}
   onMouseLeave={this.onMouseLeave}
>
```

It disables hover in web version on touch devices. Where `onMouseLeave` is not called and situations like below when component is stuck in hovered mode can happen.

screenshot 2019-01-25 at 10 06 57

File: apps/web/components/NavbarLink.js (hunk 2 above)
Comment (tbergq) [reply apps/web/components/NavbarLink.js]: Yes, agreed, however there is an eslint-rule saying `no-set-state` in componentDidMount. 

Ideally, we should have done this in the constructor, however, `next/router` can not be accessed on the server, and constructor is executed also on server.

so then we would need to change it to: 

```
this.setState({ isActive: true }); // eslint-disable-line react/no-did-update-set-state
```

**Pull request merged**